WChat/src/app/components/RootNavigator/components/styles.js

```diff
@@ -0,0 +1,28 @@
+import { StyleSheet } from 'react-native';
+import { GREY } from '../../../../utils/colors';
+import { Platform } from 'react-native'
+
+const styles = StyleSheet.create({
+  header: {
+    flexDirection: 'row'
+  },
+  headerAvatar: {
+    width: 40,
+    height: 40,
+    marginRight: 10,
+    borderRadius: 50,
+    borderWidth: 0.5,
+    borderColor: GREY,
+    alignSelf: 'center'
+  },
+  headerTitle: {
+    fontSize: Platform.OS === 'ios' ? 17 : 18,
+    fontWeight: Platform.OS === 'ios' ? '600' : '500',
```

Comment:
jajaj that's the diference between `ios` and `android`?  

`que trucazo`

Reply:
create a `config/platform.js` file and export `Platform.OS === 'ios'` as `isIos`

```
export const isIos = Platform.OS === 'ios';
export const isAndroid = Platform.OS === 'android'; 
```